console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -2547,7 +2549,7 @@ const resolvers = {
       return true;
     },
     deleteCredentialsForDevice: (parent, args) => {
-      setCredentials(args.iotproject, args.deviceId, []);
+      setCredentials(args.iotproject, args.deviceId, "");
```

Comment:
It was throwing error after delete credentials because of array []. Basically credentials is an string "[]" not array. 
Even this credentials: []  is throwing error in createIotDevice. It wil be string credentials: "".  wdyt?

  `type Device {
    deviceId: String!
    enabled: Boolean!
    viaGateway: Boolean!
    jsonData: String! #The Json representation of this device.
    credentials: String! #A Json array with the devices credentials.
  }`